python/indy-agent.py

```diff
@@ -40,6 +40,9 @@
 AGENT['ui_event_queue'] = UIEventQueue(LOOP)
 AGENT['ui_router'] = Router()
 
+AGENT['conn_router'] = Router()
```

Comment:
So I'm a little confused why you added another separate router for connection messages?

All messages that aren't UI messages should go through the main message receiver. Could you explain to me some of your reasoning here?

Reply:
I'm sorry to have caused some confusion here; I'm realizing now that I could have been more explicit. Let's talk some more about this in a call.

Reply:
@dbluhm ok